app/views/layouts/_angular_right_cell.html.haml

```diff
@@ -0,0 +1,7 @@
+= render :partial => right_cell_partial
```

Comment:
Can remove this whole file, never used..

(And putting stuff in localStorage was the original reason for my panic about the POC..  ;))